### PR TITLE
Updated Submit and Unknown buttons for Fever and Symptom Start follow up views.

### LINKED ISSUE #192: Fix button backgrounds
- Ensure symptom/submit/skip etc. button backgrounds are consistent (sizes, rounded corners) and look good in all screen sizes.
- If needed implement background programmatically (use `layer` for rounded corner and borders)
- Check Figma/ask UX re: disabled button presentation. Apply to buttons that have a disabled state (e.g. symptom selection screen when there are no symptoms selected).

### LINKED ISSUE #202: Improve layout on iPad / autolayout in general
The UI is pretty much broken on iPad. It's probably not mandatory to optimize for iPad yet, but it should be passable.

### PR DESCRIPTION
**Pull request by mellowgeek**

### CODE REVIEW

File: CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift [hunk 1]
```diff
@@ -9,9 +9,19 @@ class SymptomStartDaysViewModel {
...
 
     let setActivityIndicatorVisible: Driver<Bool>
+    
+    private var daysIsEmpty: BehaviorRelay<Bool>
+    let submitButtonEnabled: Driver<Bool>
 
     init(symptomFlowManager: SymptomFlowManager) {
         self.symptomFlowManager = symptomFlowManager
+        
+        daysIsEmpty = BehaviorRelay<Bool>(value: true)
```
Comment (mellowgeek) [inline CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift:19]: @i-schuetz is this a good way to bind the changes in the input text box to the submit button? I tried to go off of how other things were structured.

Comment (ivnsch) [reply CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift:19]: Yes 👍

Comment (mellowgeek) [reply CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift:19]: Great, I'll go and implement this in the other views with this pattern.

File: CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift [hunk 2]
```diff
@@ -9,9 +9,19 @@ class SymptomStartDaysViewModel {
     let title = L10n.Ux.Symptomsdays.heading
 
     let setActivityIndicatorVisible: Driver<Bool>
+    
+    private var daysIsEmpty: BehaviorRelay<Bool>
```
Comment (ivnsch) [inline CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift]: You can make this `let`

File: CoEpi/ui/onboarding/OnboardingViewController.swift [hunk 3]
```diff
@@ -100,6 +100,10 @@ class OnboardingViewController: UIViewController {
         joinButtonLabel.setTitle(L10n.Ux.Onboarding.join, for: .normal)
         questionsButtonLabel.setTitle(L10n.Ux.Onboarding.questions, for: .normal)
         
+        ButtonStyles.applyStyleUnselectedButton(to: joinButtonLabel)
```
Comment (ivnsch) [inline CoEpi/ui/onboarding/OnboardingViewController.swift]: Let's name these methods just `applyUnselected` (`applyX(to:)` in general)?

Comment (mellowgeek) [reply CoEpi/ui/onboarding/OnboardingViewController.swift]: Done

**Review by ivnsch — APPROVED** [review]

**Pull request merged**